app/decorators/application_choice_export_decorator.rb

```diff
@@ -57,6 +57,6 @@ def summary_for_gcse(gcse)
     return if gcse.blank?
 
     qualification = ApplicationQualificationDecorator.new(gcse)
-    "#{qualification.qualification_type.humanize} #{qualification.subject.capitalize}, #{qualification.grade_details.join(' ')}, #{qualification.start_year}-#{qualification.award_year}"
```

Comment:
Looking through prod data, we do have some gcse start year's populated, maybe we should account for both it being there vs not being there?